### DISCUSSION TITLE
How to Build the project?

### DISCUSSION THREAD

**Discussion by github-gsjb** [LSPosed/LSPosed#1099] [Q&A] [answered]
How to Build  to .zip file ?

**Comment by yujincheng08** [comment, marked as answer]
https://github.com/LSPosed/LSPosed/blob/a29d08a4eb8ea0abcd0367ee07c2f78ab3eea206/.github/workflows/core.yml#L80

**Reply by github-gsjb** [reply]
> https://github.com/LSPosed/LSPosed/blob/a29d08a4eb8ea0abcd0367ee07c2f78ab3eea206/.github/workflows/core.yml#L80

Thank you

**Reply by Hacker-rep** [reply]
Thank you

**Comment by fahime-ghasemi** [comment]
I've used zipZygiskRelease task to build a zip file but I don't know where the zip file is located. I can't find it. Could you please help me?

**Reply by mywalkb** [reply]
You can find them in magisk-loader/release/, there will be 4 zip: relaese and debug for riru and zygisk
